tickit/devices/eiger/eiger_settings.py

```diff
@@ -0,0 +1,154 @@
+from dataclasses import dataclass, field, fields
+from enum import Enum
+from typing import Any, List
+
+from .eiger_schema import (
+    AccessMode,
+    field_config,
+    ro_float,
+    ro_str,
+    rw_bool,
+    rw_float,
+    rw_int,
+    rw_str,
+)
+
+FRAME_WIDTH: int = 4148
+FRAME_HEIGHT: int = 4362
+
+
+class KA_Energy(Enum):
+    """Possible element K-alpha energies for samples."""
+
+    Li: float = 54.3
+    Be: float = 108.5
+    B: float = 183.3
+    C: float = 277.0
+    N: float = 392.4
+    O: float = 524.9
+    F: float = 676.8
+    Ne: float = 848.6
+    Na: float = 1040.98
+    Mg: float = 1253.6
+    Al: float = 1486.7
+    Si: float = 1739.98
+    P: float = 2013.7
+    S: float = 2307.84
+    Cl: float = 2622.39
+    Ar: float = 2957.7
+    K: float = 3313.8
+    Ca: float = 3691.68
+    Sc: float = 4090.6
+    Ti: float = 4510.84
+    V: float = 4952.2
+    Cr: float = 5414.72
+    Mn: float = 5898.75
+    Fe: float = 6403.84
+    Co: float = 6930.32
+    Ni: float = 7478.15
+    Cu: float = 8047.78
+    Zn: float = 8638.86
+    # TODO: Add more elements?
```

Comment:
See [https://github.com/dls-controls/tickit/pull/25/files#r742072717](https://github.com/dls-controls/tickit/pull/25/files#r742072717)

Reply:
"Resolve or remove this" - me